lib/msf/core/post/linux/system.rb

```diff
@@ -270,6 +300,110 @@ def get_mount_path(filepath)
   end
 
 
+  #
+  # Gets all the IP directions of the device
+  # @return [Array]
+  #
+  def ips()
+    lines = read_file("/proc/net/fib_trie")
+    result = []
+    previous_line = ""
+    lines.each_line do |line|
+      if line.include?("/32 host LOCAL")
+        previous_line = previous_line.split("-- ")[1].strip()
+        if not result.include? previous_line
+          result.insert(-1, previous_line)
+        end
+      end
+      previous_line = line
+    end
+    return result
+  end
+
+
+  #
+  # Gets all the interfaces of the device
+  # @return [Array]
+  #
+  def interfaces()
```

Comment:
Style. No need to use brackets `()` in method definitions when the method takes no arguments.